src/github_review_prompts/prompt_generator.py

```
@@ -0,0 +1,435 @@
+            head_branch = pr_info.get('head_branch', '')
+            
+            if head_repo and head_branch:
+                lines.extend([
+                    "**🔄 作業開始コマンド**:",
+                    "```bash"
+                ])
+                
+                # 同じリポジトリかフォークかで分岐
+                if head_repo == base_repo:
+                    lines.extend([
+                        f"# ローカルでソースブランチにチェックアウト",
```

Ruff (0.12.2)

f-string without any placeholders

Remove extraneous `f` prefix

(F541)